zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js

```diff
@@ -129,6 +142,7 @@ const styles = theme => ({
     },
     '& svg': {
       fill: 'white',
+      justifyContent:'Space-arround'
```

Comment:
@kamthamc 
I should remove the values?

Reply:
Yes please, please use `space-around` if it works/ fixes the problem other wise remove it

Reply:
> Yes please, please use `space-around` if it works/ fixes the problem other wise remove it

Hello @kamthamc 
I have done the changes and after pulling the updates from the repo for days now have being having issues with starting up the docker containers. 
Currently what am doing is i reset and clean the docker container and am trying to start again boss.
that is why changes that was requested i have not yet create a PR

Reply:
> Yes please, please use `space-around` if it works/ fixes the problem other wise remove it

I already did the changes and when trying to run to see updates i started errors messages like unable to fetch database

Reply:
> Yes please, please use `space-around` if it works/ fixes the problem other wise remove it

Hello @kamthamc 
is not working
![zublip](https://user-images.githubusercontent.com/70459251/163702268-07faf385-c5c3-499d-8058-abbf10e4b6b0.png)

Reply:
@godsakani i did mention the correct value of space-around. I didn’t say it would fix the problem you’re trying to solve. If its a just consistent spacing try `gap` css property

Reply:
> @godsakani i did mention the correct value of space-around. I didn’t say it would fix the problem you’re trying to solve. If its a just consistent spacing try `gap` css property

Ok Sir @kamthamc 
thank you . I noticed it had no changed on it

Reply:
The property is valid for flexbox/grid did you try adding it to the flexbox itself?